pyaccel/new_lifetime.py

```diff
@@ -17,6 +19,8 @@
 bunch_lengh = 2.3e-3 #m
 coupling = 0.01
 energy_accep = 0.03
+ring_loc = 0
+
 
 
 
```

Comment:
The same is valid here for the lattice definition.

Also, I think the lines: 

```
J = pyaccel.optics.EquilibriumParameters.Je #damping factors
E_n = pyaccel.optics.EquilibriumParameters.emit0#naturalEmittance;
tau_am = pyaccel.optics.EquilibriumParameters.taue #radiationDamping;
```
will not work as intended, because `EquilibriumParameters` is a python class.
In order for it to work properly, you have to instantiate an object, passing the lattice model as input:
```
eqpars = EquilibriumParameters(lattice)
eqpars.Je
eqpars.emit0
```